lib/jekyll/strapi/generator.rb

```diff
@@ -25,11 +25,10 @@ def initialize(site, base, document, collection)
         @dir = @collection.config['output_dir'] || collection.collection_name
         # Default file name, can be overwritten by permalink frontmatter setting
         @name = "#{document.id}.html"
+        filename_to_read = File.join(base, "_layouts"), @collection.config['layout']
 
         self.process(@name)
         self.read_yaml(File.join(base, "_layouts"), @collection.config['layout'])
-
-        # Use the permalink collection setting if it is set
```

Comment:
Why remove this comment? This PR is for authorization only, other changes should have new PR.

Reply:
Without other changes this PR would not have sense. It addressed working API with Strapi 4.

Then, what we are really missing - some statement from Strapi guys what are plans to do with repository - since they own this one. Or they need to put someone in charge to maintain project, or release repo, or let us know that they killed it so we can fork to other repo, rebrand and publish gems ;)